e2e/cypress/integration/system_console/team_and_channel_assign_roles_spec.js

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Since this testcase is related to an Enterprise feature, please move this testcase to the folder
`e2e/cypress/integration/enterprise/system_console`